Update mitsubishi2mqtt.ino (fixed widevane error in compilation)

mamrai1 opened the pull request:
1.fixed widevane error in compilation
2. removed dublicate entry of widevane mode 6

mamrai1 committed:
Update mitsubishi2mqtt.ino

mamrai1 committed:
Update mitsubishi2mqtt.ino

missing ')'

melyux commented:
Looks good! Fixes the typo. Disregard my push to my branch with the same thing.

melyux reviewed (changes requested):
@mamrai1 In line 1288, `const size_t bufferSizeInfo = JSON_OBJECT_SIZE(7);` needs to be changed to 8 instead of 7, since we added the wideVane key. Otherwise I've found that it cuts off the last key (in this case, "mode").

mamrai1 committed:
fix json

const size_t bufferSizeInfo = JSON_OBJECT_SIZE(7); > const size_t bufferSizeInfo = JSON_OBJECT_SIZE(8);

mamrai1 committed:
Update mitsubishi2mqtt.ino

add to fix HP turn off after change temperature

gysmo38 commented:
Thank you

melyux commented on src/mitsubishi2mqtt/mitsubishi2mqtt.ino:
```diff
@@ -1408,7 +1408,16 @@ void mqttCallback(char* topic, byte* payload, unsigned int length) {
   else if (strcmp(topic, ha_temp_set_topic.c_str()) == 0) {
+    //add to fix HP turn off after change temperature
+    heatpumpSettings currentSettings = hp.getSettings();
+    delay(10);
```
Was there a reason for this naked 10 second delay? Seems kind of drastic!

lesleypersyn replied:
Syntax
delay(ms)
ms: the number of milliseconds to pause

melyux replied:
Doh. I think he was going for 10 seconds though

mamrai1 commented:
Maybe 7 sec would be enough?

On Fri, 18 Sep 2020, 08:45 melyux, <notifications@github.com> wrote:

> *@melyux* commented on this pull request.
> ------------------------------
>
> In src/mitsubishi2mqtt/mitsubishi2mqtt.ino
> <https://github.com/gysmo38/mitsubishi2MQTT/pull/101#discussion_r490715124>
> :
>
> > @@ -1408,7 +1408,16 @@ void mqttCallback(char* topic, byte* payload, unsigned int length) {
>      hp.update();
>    }
>    else if (strcmp(topic, ha_temp_set_topic.c_str()) == 0) {
> +    //add to fix HP turn off after change temperature
> +    heatpumpSettings currentSettings = hp.getSettings();
> +    delay(10);
>
> Was there a reason for this naked 10 second delay? Seems kind of drastic!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/gysmo38/mitsubishi2MQTT/pull/101#pullrequestreview-491165928>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANVCNGH7RHLNJFHQE7MVL4DSGLXW7ANCNFSM4QWALPFA>
> .
>

melyux commented:
But what is the purpose of the delay?

melyux commented:
That delay doesn't seem to fix the turning off issue. Mine turns off anyway by itself sometimes.
